docs/modules/deploy/partials/nav.adoc

```diff
@@ -16,6 +16,6 @@
 *** xref:deploy:deploying-in-kubernetes.adoc[Overview]
 *** xref:deploy:configuring-kubernetes.adoc[]
 *** xref:deploy:scaling-and-upgrading-kubernetes.adoc[]
-*** xref:deploy:deploying-with-operator.adoc[]
+*** xref:operator:root:index.adoc[Hazelcast Platform Operator]
```

Comment:
I didn't get the reason for this change? "Hazelcast Platform Operator" in this section doesn't point to anywhere.